Remove members from About (#1342)
publiccodenet/about: activities/creating-affiliations/affiliate-template.md

[BLURB ABOUT ORGANIZATION]

> The Foundation for Public Code is a non-profit association with as mission to enable public-purpose software and policy that is open and collaborative.

Reviewer: ```suggestion
The Foundation for Public Code is a non-profit association whose mission is to enable public-purpose software and policy that is open and collaborative.
```
